src/ebpf.rs

```diff
@@ -51,6 +51,14 @@ pub const MM_STACK_START: u64 = MM_REGION_SIZE * 2;
 pub const MM_HEAP_START: u64 = MM_REGION_SIZE * 3;
 /// Virtual address of the input region
```

Comment:
shouldn't "Input" be renamed?

Reply:
I did not rename it, because it is still the same. Should it be called `MM_TX_INPUT_START`?

Reply:
"Input" is kinda a useless name because it does not say from where to where, it is subjective depending on the perspective, like "left" and "right". Maybe `MM_TX_AREA`, `MM_TX_INSTRUCTION_AREA` and `MM_TX_INSTRUCTION_DATA`.